### PR TITLE
PLAT-516 refresh configs when UP

### PR DESCRIPTION
**Pull request by nour-borgi**
Fixes: 
* Fix the "up" with refreshing the superset and kafka mapper consumer configs
* Fix the "destory": updating the labels to match the name of the labels on the platform so a destory can remove the old configs 
* Hot fix in the superset dashboard [updating type]

Needs to include the new changes of hapi-fhir config importer..

### CODE REVIEW

**Review by michaelloosen — APPROVED** [review]
LGTM

**Review by michaelloosen — DISMISSED** [review]

**Review by Lumiere-Lodi — DISMISSED** [review]
You just need to fix the conflicts with main.

**Review by Lumiere-Lodi — APPROVED** [review]
LGTM

File: package-metadata.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "dependencies": [
     "kafka-unbundler-consumer",
     "kafka-mapper-consumer",
+    "fhir-datastore-hapi-fhir",
```
Comment (michaelloosen) [inline package-metadata.json:10]: This doesn't seem right, why do we need to include hapi-fhir in cares?

**Review by michaelloosen — APPROVED** [review]
LGTM

**Pull request merged**